examples/step-19/doc/results.dox

```diff
@@ -0,0 +1,290 @@
+  Total number of particles in simulation:  44
+  Number of particles lost this time step:  6
+  Fraction of particles lost through anode: 0.0601266
+
+  Now at t=4.93276e-05, dt=4.87463e-08.
+
+Timestep 1001
+  Field degrees of freedom:                 4989
+  Total number of particles in simulation:  44
+  Number of particles lost this time step:  0
+  Fraction of particles lost through anode: 0.0601266
+
+  Now at t=4.93759e-05, dt=4.82873e-08.
+
+
+...
+
+
+Timestep 2091
+  Field degrees of freedom:                 4989
+  Total number of particles in simulation:  44
+  Number of particles lost this time step:  0
+  Fraction of particles lost through anode: 0.0503338
+
+  Now at t=9.99237e-05, dt=4.26254e-08.
+
+Timestep 2092
+  Field degrees of freedom:                 4989
+  Total number of particles in simulation:  44
+  Number of particles lost this time step:  0
+  Fraction of particles lost through anode: 0.0503338
+
+  Now at t=9.99661e-05, dt=4.24442e-08.
+
+Timestep 2093
+  Field degrees of freedom:                 4989
+  Total number of particles in simulation:  44
+  Number of particles lost this time step:  2
+  Fraction of particles lost through anode: 0.050308
+
+  Now at t=0.0001, dt=3.38577e-08.
+```
+
+Picking a random few time steps, we can visualize the solution in the
+form of streamlines for the electric field and dots for the electrons:
+<div class="twocolumn" style="width: 80%">
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step-19.solution.0000.png"
+         alt="The solution at time step 0 (t=0 seconds)."
+         width="500">
+    <br>
+    Solution at time step 0 (t=0 seconds).
+    <br>
+  </div>
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step-19.solution.1400.png"
+         alt="The solution at time step 1400 (t=0.000068 seconds)."
+         width="500">
+    <br>
+    Solution at time step 1400 (t=0.000068 seconds).
+    <br>
+  </div>
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step-19.solution.0700.png"
+         alt="The solution at time step 700 (t=0.000035 seconds)."
+         width="500">
+    <br>
+    Solution at time step 700 (t=0.000035 seconds).
+    <br>
+  </div>
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step-19.solution.2092.png"
+         alt="The solution at time step 2092 (t=0.0001 seconds)."
+         width="500">
+    <br>
+    Solution at time step 2092 (t=0.0001 seconds).
+    <br>
+  </div>
+</div>
+
+That said, a more appropriate way to visualize the results of this
+program are by creating a video that shows how these electrons move, and how
+the electric field changes in response to their motion:
+
+@htmlonly
+<p align="center">
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/HwUtE7xuteE"
+   frameborder="0"
+   allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+   allowfullscreen></iframe>
+ </p>
+@endhtmlonly
```

Comment:
Would it be possible to add the time to the movie using the annotate time filter of Paraview? If you output a pvd file + use this filter it could help the discussion I believe (just my opinion though).

Reply:
I played with this. It just looks so bulky. From an artistic perspective, I found that really unappealing :-(